lib/ui/search_ol_editions_screen/search_ol_editions_screen.dart

```diff
@@ -192,6 +194,7 @@ class _SearchOLEditionsScreenState extends State<SearchOLEditionsScreen> {
                             onPressed: () => _saveEdition(
                               result: item,
                               cover: item.covers![0],
+                              work: item.works![0].key,
```

Comment:
I think this is not null safe. What do you think about:
`work: item.works != null && item.works!.isNotEmpty ? item.works![0].key : null,`